src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -276,6 +276,7 @@ def load_arguments(self, _):
         c.argument('vm_name', name_arg_type, id_part=None, help='Name of the virtual machine.', completer=None)
         c.argument('os_disk_size_gb', type=int, help='the size of the os disk in GB', arg_group='Storage')
         c.argument('availability_set', help='Name or ID of an existing availability set to add the VM to. None by default.')
+        c.argument('vmss', help='Virtual machine scale set that the virtual machine should be assigned to.')
```

Comment:
Name or ID of Virtual machine scale set that the virtual machine should be assigned to.

Reply:
> 
> 
> Name or ID of Virtual machine scale set that the virtual machine should be assigned to.

Good